setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup_args = {}
 install_requires = [
-    "param >=1.9.3,<2.0",
+    "param >=1.12.0,<2.0",
```

Comment:
Shouldn't the upper pin be removed?

Reply:
Yep I guess we'll need to update the upper pin soon, to get HoloViews (and others) to be tested with alpha versions of Param. I'll update it to `<3.0`.